src/client/components/dimension-actions-menu/dimension-actions-menu.tsx

```diff
@@ -71,7 +71,7 @@ export const DimensionActions: React.SFC<DimensionActionsProps> = (props: Dimens
 
   const hasSplitOn = splits.hasSplitOn(dimension);
   const isOnlySplit = splits.length() === 1 && hasSplitOn;
-  const isStringDimension = dimension.kind === "string";
+  const isPinable = dimension.kind === "string" || dimension.kind === "boolean";
```

Comment:
Nice! Is it feasible to define isPinable property/function inside dimension?

Reply:
Pinnable type class would be ideal. I don't like idea that dimension knows anything about "pin" action.

Reply:
good point, but I would encapsulate "string" and "boolean" at least, e.g dimension.isString dimension.isBoolean